.github/workflows/rust-test-with-style.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        compiler: [clang]
+        compiler: [gcc-10]
```

Comment:
Do you still want this change?

Reply:
I'm fine either way; I don't think it makes any difference. I removed it in this commit because it was originally a debugging commit that slipped into main.